SplunkAppForWazuh/appserver/static/js/controllers/management/cdb/managerCdbCtrl.js

```diff
@@ -59,6 +59,11 @@ define([
      * On controller load
      */
     $onInit() {
+      // Data validation
+      this.scope.nameValidationRegex = '^[\\w\\-]+$'
+      this.scope.keyValidationRegex = '(?:^"([\\w\\-:]+?)"|^[^:"\\s]+$)'
+      this.scope.valueValidationRegex = '(?:^"([\\w\\-:]*?)"$|^[^:"\\s]*$)'
```

Comment:
**question**: where do these regular expressions come from?

Reply:
From the Wazuh API --> https://github.com/wazuh/wazuh/blob/eaab105fb5a7430173a52a2598deef204b06e809/framework/wazuh/core/cdb_list.py#L185

Reply:
**nitpick**: Check the regex, the `valueValidationRegex` is not the equal. See https://github.com/wazuh/wazuh/blob/v4.3.0-rc4/framework/wazuh/core/cdb_list.py#L203. The regex in the app invalidates the `values` with spaces at least.

Reply:
Solved